components/blocks/About.vue

```diff
@@ -67,20 +67,27 @@
             :name="person.name"
             :title="person.title"
             :team="person.team"
-            :main-image="
-              'https://ccv.brown.edu/images/people/JPEG/' + person.image
-            "
+            :main-image="'/static/images/people/' + person.image"
```

Comment:
the `static` isn't needed here - things in the static folder get move to public so the path should be `/images/people`